pkg/docker/docker.go

```diff
@@ -619,19 +619,22 @@ func LoginToRegistry(address string, username string, password string) *DockerEr
 	// Pipe the password via stdin.
 	cmd := exec.Command("docker", "login", "--username", username, "--password-stdin", address)
 	stdin, err := cmd.StdinPipe()
+	textDockerLoginFailed := fmt.Sprintf("Docker login to %s as %s failed. Address, username or password is incorrect", address, username)
 	if err != nil {
-		return &DockerError{errDockerCredential, err, "Error executing 'docker login'"}
+		loginErr := errors.New(textDockerLoginFailed)
+		return &DockerError{errDockerCredential, loginErr, textDockerLoginFailed}
```

Comment:
Not sure if it would be better to send the original error `err` and then the text here so we can debug this easier?

Reply:
That's the cause of the original issue - err returned from `os/exec` is `exit status 1` for the example inputs. As this is the error message displayed when the `--json` flag is set, that's what is seen in the IDEs.

I'm not sure of a way to include both our custom error, and the original `os/exec` error in the outputs.